analytics_data_api/v0/apps.py

```diff
@@ -1,28 +1,6 @@
 from django.apps import AppConfig
-from django.conf import settings
-from elasticsearch_dsl import connections
```

Comment:
can we strip elasticsearch support out of requirements?

Reply:
Just ripped it out!